lib/logger.c

```diff
@@ -103,16 +103,17 @@ void log_log(int level, const char *file, int line, const char *ns,
       // va_list can only be consumed once. Make a copy so we can use it again
       va_list copy;
       va_copy(copy, args);
-
+      size_t linedFmt_length = strlen(fmt) +
```

Comment:
Why are we changing the code here and why are we allocating on heap now as opposed to on the stack?